src/main/java/gndata/lib/util/change/Merge.java

```diff
@@ -0,0 +1,65 @@
+package gndata.lib.util.change;
+
+import java.util.*;
+
+import com.hp.hpl.jena.ontology.OntModel;
+import com.hp.hpl.jena.rdf.model.*;
+import com.hp.hpl.jena.vocabulary.*;
+
+/**
+ * Class that knows how to update Metadata entities.
+ */
+public class Merge implements Change {
+
+    private Model new_object;
+    private OntModel ontology;
+    private boolean applied = false;
+
+    private Model changes = ModelFactory.createDefaultModel();
+
+    public Merge(Model new_object, OntModel ontology) {
+        this.new_object = new_object;
+        this.ontology = ontology;
+    }
+
+    public void applyTo(Model m) {
```

Comment:
Maybe it is better to change the signature to:

```java
public void applyTo(Model annotations, Model ontology);
```

Because also other Change implementations may need an ontology and then you would not have store a reference in all these classes.